presto-docs/src/main/sphinx/release/release-0.288.rst

```diff
@@ -0,0 +1,86 @@
+* Add support for memoizing in resource group state info endpoint. This can be enabled by setting ``cluster-resource-group-state-info-expiration-duration`` to a non-zero duration. :pr:`22764`
+* Add support for non default keystore and truststore type in presto CLI and JDBC. :pr:`22556`
+* Add support for querying system.runtime.tasks table in native clusters. :pr:`21416`
+* Remove deprecated feature and configuration property ``deprecated.group-by-uses-equal``, which allowed group by to use equal to rather than distinct semantics. :pr:`22888`
+* ... Improve :doc:`/presto-cpp` documentation. :pr:`22717`
+* Change error code for cast from DOUBLE or REAL to BIGINT, INTEGER, SMALLINT or TINYINT for out of range values from ``NUMERIC_VALUE_OUT_OF_RANGE`` to ``INVALID_CAST_ARGUMENT``. :pr:`22917`
+* Change handling of floating point numbers in Presto to consistently treat NaNs as larger than any other number and equal to itself.  It also changes the handling of positive and negative zero to always be considered equal to each other.  Read more here: https://github.com/prestodb/rfcs/blob/main/RFC-0001-nan-definition.md. The new nan behavior can be disabled by setting the configuration property ``use-new-nan-definition`` to ``false``. This configuration property is intended to be temporary to ease migration in the short term, and will be removed in a future release. :pr:`22386`
+* Enable HBO for CTE materialized query. :pr:`22606`
+* Prestissimo support for CTAS into bucketed (but not partitioned) tables. pr:`22737`
+* Update CI pipeline to build and publish native worker docker image. :pr:`22806`
+* Upgrade Alluxio to 313. :pr:`22958`
+* Upgrade io.jsonwebtoken artifacts to 0.11.5. :pr:`22762`
+* Upgrade fasterxml.jackson artifacts to 2.11. :pr:`22417`
+
+Hive Connector Changes
+______________________
+* Improve affinity scheduling granularity from a file to a section of a file by adding a ``hive.affinity-scheduling-file-section-size`` configuration property and ``affinity_scheduling_file_section_size`` session property. The default file size is 256MB. :pr:`22563`
+
+Iceberg Connector Changes
+_________________________
+* Improve the partition specs that must be checked to determine if the partition supports metadata deletion or predicate thoroughly pushdown. :pr:`22753`
+* Improve time travel ``TIMESTAMP (SYSTEM_TIME)`` syntax to include timestamp-with-time-zone data type. :pr:`22851`
+* Improve time travel ``VERSION (SYSTEM_VERSION)`` syntax to include snapshot id using `BIGINT` data type. :pr:`22851`
+* Add procedure `expire_snapshots` to remove old snapshots in Iceberg. :pr:`22609`
+* Add support for Iceberg REST catalog. :pr:`22417`
+* Add time travel ``BEFORE`` syntax for Iceberg tables to return historical data. :pr:`22851`
+* Disable timestamp with time zone in ``CREATE``, ``ALTER``, and ``INSERT`` statements. :pr:`22926`
```

Comment:
```suggestion
* Remove timestamp with time zone in ``CREATE``, ``ALTER``, and ``INSERT`` statements. :pr:`22926`
```
Minor rephrasing following the [Order of Changes](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes) in the Release Notes Guidelines.